velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -110,8 +110,20 @@ class Re2FunctionsTest : public test::FunctionBaseTest {
             std::optional<std::string>{input});
       };
 
+      // Print the failing pattern to make it easier to locate the failed case.
```

Comment:
May use SCOPED_TRACE: https://github.com/google/googletest/blob/main/docs/advanced.md#adding-traces-to-assertions